libpermian/workflows/grouped.py

```diff
@@ -61,6 +63,8 @@ def run(self):
         try:
             self.setup()
             self.execute() if not self.dryRun else self.dry_execute()
+        except self.silent_exceptions as e:
+            self.groupLog(f'Workflow failed with silent exception: {e}')
```

Comment:
Even though it's a silent exception I believe it should still report DNF/ERROR final result. Especially if the exception is raised in setup.

Reply:
Also instead of "failed" it would be better to use either "hit" or "raised" as "failed" may sound like it's related to the test result.

Reply:
The DNF/ERROR result is handled elsewhere in the pipeline, but I see why it is better to explicitly report it here.